crates/core/component/dex/src/lp/position.rs

```diff
@@ -214,6 +214,61 @@ impl Position {
             asset_id: self.phi.pair.asset_2(),
         }
     }
+
+    /// Compute the flows compared to a previous position
+    pub fn flows(&self, prev: &Self) -> Flows {
+        Flows::from_fee_and_reserves(self.phi.component.fee, &self.reserves, &prev.reserves)
+    }
+}
+
+#[derive(Clone, Copy, Debug)]
+pub struct Flows {
```

Comment:
I think this struct is a good idea, but fwiw its current shape seems informed by the previous implementation, but I think there's an opportunity to do something better here. The position execution recording in dex_ex has a context start/end, this makes it slightly confusing to use because the delta/lambda can be of totally different asset types than start/end (e.g, multi-hop path).

Instead, it would be useful to be able to address each position execution uniquely, and index these identifiers by context start/end, or undirected trading pair, or LQT eligible activity etc. 

Since this is already a useful refactor we can keep it like this, not blocking.